src/ripple/protocol/Feature.h

```diff
@@ -34,6 +34,52 @@ uint256
 feature (const char* name);
 /** @} */
 
+constexpr size_t FeatureBitsetSize = 32;
```

Comment:
I agree that we want to make it hard to miss the situation when the number of features exceeds this limit. With the current assert, here is what you see when you start `rippled`:

```
Assertion failed: (features.size() < FeatureBitsetSize), function feature, file /Users/ellery/work/rippled/src/ripple/protocol/impl/Feature.cpp, line 57.
Abort trap: 6
```
I guess the only concern with that is that you won't hit it if assertions are disabled, but I think we generally build with asserts for CI and local, so this should be low risk? 

That said, I have no problem with the exception option either.